dev-docs/bidders/1accord.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: bidder
+title: 1accord
+description: Prebid 1accord Bidder Adapter
+pbjs: true
+biddercode: 1accord
+aliasCode : nexx360
+gvl_id: 965
+tcfeu_supported: true
+usp_supported: true
+gpp_supported: true
+schain_supported: true
+dchain_supported: false
+floors_supported: true
+userIds: all
+tcf2_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+sidebarType: 1
+
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description                | Example                                   | Type      |
+|---------------|----------|----------------------------|--------------------------------------     |-----------|
+| `tagId`       | required | tag ID                     | `"testnexx"`                              | `string`  |
+
+### First Party Data
+
+Publishers should use the `ortb2` method of setting [First Party Data](https://docs.prebid.org/features/firstPartyData.html).
```

Comment:
This is good, but you might consider adding detail about what specific data fields are supported: , site.data, user.data, site.ext.data, user.ext.data, imp.ext.data.

If you do that, then you can add a metadata field above `fpd_supported: true`

Reply:
Done